### ISSUE TITLE
No text alternatives for keyboard focus

### ISSUE THREAD

**Issue by JoelHaslam** [grafana/grafana#39784]
The narrator tool works very well with the UI when using only a keyboard, however there no alternative text appears for any non text content on the web page if they have the keyboard focus.

**Steps to reproduce**:
Load up a grafana web page, an hit tab until the keyboard focus moves to any of the icons (that produce text if you hover over them with a mouse)

**Actual Result**:
No text appears.
![image](https://user-images.githubusercontent.com/63413067/135280326-9d413b98-3ff7-48fd-8d4f-255db16cface.png)

**Expected Result**:
Text to appear that is the equivalent of hovering over with a mouse.
![image](https://user-images.githubusercontent.com/63413067/135280411-b8cf1a50-d941-4045-9661-6437aa6b51fc.png)

**Relevant WCAG Criteria:** [1-1-1 non-text-content](https://www.w3.org/TR/WCAG21/#x1-1-1-non-text-content)

**Environment**:
- Grafana version: 8.1.4
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by torkelo** [comment]
seems like native tooltips don't show either on focus, screen readers should work fine have aria labels
